packages/shared/src/lib/analytics.ts

```diff
@@ -114,6 +114,9 @@ export enum AnalyticsEvent {
   DeclineContentScripts = 'decline content scripts',
   ToggleFeedPreview = 'toggle feed preview',
   CreateFeed = 'create feed',
+  // generic referral - start
+  AcceptGeneralReferralInvitation = 'accept generic referral invitation',
```

Comment:
Still not confirmed. Conversation in thread: https://dailydotdev.slack.com/archives/C05P9ET7S9K/p1698942032420289

Reply:
The taxonomy is different, also replied on the thread.